corona-warn-app/cwa-app-android
# Contact Diary data is written in clear text into error log on data deletion (EXPOSUREAPP-7408)

### Description
We have this problem that some of our censors read the data that needs to be censored directly from the database. However, when something is logged after the respective data is already removed, the censor can't censor anymore. 

This PR now updates these censors so that they keep the data in memory. Then, even after the data is removed from the database, we can still censor correctly. 

Actually, we had some tests that covered these scenarios, however, they were incorrect (false positive). 

### How to test
Delete
- Single Persons in the Contact Diary
- Single Locations in the Contact Diary
- CheckIns
- TraceLocations
=> Then export the error logs and check if all data got censored appropriately.
